tests/queries/0_stateless/02442_auxiliary_zookeeper_endpoint_id.sql

```diff
@@ -0,0 +1,21 @@
+-- Tags: no-fasttest
```

Comment:
Seems that the lastest failures have something to do with the long time the test case (02442_auxiliary_zookeeper_endpoint_id) takes.  Is there any way to shorten the test case run time?
Here is the error info I found:
`2023-05-24 18:02:28 02442_auxiliary_zookeeper_endpoint_id:                                  [ FAIL ] 236.52 sec. - Test runs too long (> 60s). Make it faster`

Reply:
This error can be ignored, and it's absolutely irrelevant to the `no-fasttest` tag, let's return the tag back.